docs/tutorials/ofo.md

```diff
@@ -0,0 +1,113 @@
+- [k9s](https://k9scli.io/) (optional, if you'd like to inspect your cluster visually)
+
+### Let's see the code!
+
+<!-- TODO: add/remove this based on PR feedback -->
+ATTN REVIEWIERS: Should we include some code snippets here? or is it too much?
+
+### Show me the commands!
+
+#### Build our cluster with kind
+
+OK, let's get our cluster up and running! If you already have a K8s cluster, you can skip to [Install cert-manager](#install-cert-manager).
+
+First, create our cluster using the `kind-cluster.yaml` file:
+```shell
+kind create cluster --config kind-cluster.yaml
+```
+This might take a minute or two.
+
+#### Install cert-manager
+
+Great! Next, because our operator makes use of webhooks, we need some certificate infrastructure in our cluster.
+If your cluster already has cert manager, or you're using another solution for certificate management, you can skip to [Create Namespace](#create-namespace).
+
+Install cert-manager, and wait for it to be ready: 
+```shell
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml && \
+kubectl wait --timeout=60s --for condition=Available=True deploy --all -n 'cert-manager'
+```
+
+#### Create namespace
+
+Next, we need to create a namespace for the operator and our workload: 
+```shell
+kubectl create namespace open-feature-operator-system
+```
+
+#### Install OpenFeature operator certificate
+
+Now let's install the Openfeature operator certificate: 
+```shell
+kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.2.5/certificate.yaml
+```
+
+#### Install OpenFeature operator
+
+And finally, let's install the operator itself:
+```shell
+kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.2.5/release.yaml && \
+kubectl wait --timeout=60s --for condition=Available=True deploy --all -n 'open-feature-operator-system'
+```
+
+#### Deploy our workload
+
+Now that the operator is ready to go, we can deploy our workload:
+
+```shell
+kubectl apply -f end-to-end.yaml && \
+kubectl wait --timeout=60s deployment --for condition=Available=True -l 'app=open-feature-demo'
+```
+
+Now you should see our fictional app at http://localhost:30000
+
+If you're using `k9s` or some other means of visualization, your cluster should look something like this:
+
+![k9s](@site/static/img/tutorials/k9s.png)
+
+### Experiment with OpenFeature
+
+For this demo, we get flag definitions from the custom resource definition you applied to K8s above (`end-to-end.yaml`).
+You can modify the flag values in the `featureFlagSpec` and reapply the CRD to see the changes.
+This file also contains service and deployment definitions, but these need not be modified as part of this demo.
+You may be interested the `openfeature.dev*` annotations though, which the OpenFeature operator uses to detect which workloads require flagd.
+
+Let's get started learning how OpenFeature is helping Fib3r manage this landing page!
```

Comment:
```suggestion
```shell
# Lets grab the end-to-end.yaml so that we can modify it
 curl -sfL https://github.com/open-feature/open-feature-operator/releases/download/v0.2.5/end-to-end.yaml > end-to-end.yaml
```
```